core/migrations/0011_auto_20180710_1540.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.0.1 on 2018-07-10 15:40
```

Comment:
Looks like you generated several migrations over the course on adding the `purchase_date` field. While checking all of them in still works fine at the system level, I have often found that generating one migration file for every change on the models results in a more readable diff. Having one file to review when there are changes affecting the database means that the team has a better understanding of the implication of the changes. You could delete<sup>1</sup> the three migration files (`0009_asset_purchase_date.py`, `0010_auto_20180710_1244.py` and `0011_auto_20180710_1540.py`) and generate one consolidated file.

1. **Remember** to commit the deletions

Reply:
@collinmutembei thanks, changes effected.